forge-gui/res/cardsfolder/upcoming/march_of_progress.txt

```diff
@@ -0,0 +1,8 @@
+Name:March of Progress
+ManaCost:2 U
+Types:Sorcery
+A:SP$ CopyPermanent | Cost$ 2 U | ValidTgts$ Artifact.Creature+YouCtrl | TgtPrompt$ Select target artifact creature you control | SpellDescription$ Choose target artifact creature you control. For each creature chosen this way, create a token that’s a copy of it.
+A:SP$ CopyPermanent | Cost$ 6 U | PrecostDesc$ Overload | CostDesc$ {6}{U} | Defined$ Valid Artifact.Creature+YouCtrl | NonBasicSpell$ True | SpellDescription$ (You may cast this spell for its overload cost. If you do, change its text by replacing all instances of "target" with "each.") | StackDescription$ Create a token of each artifact creature you control. 
```

Comment:
I see what you're doing with this StackDesc... but it should probably just be: `Choose each artifact creature you control. For each creature chosen this way, create a token that's a copy of it.`

Reply:
The wording on this card is just awful. Why not: "Create a token copy of target artifact creature you control." ? 🤦‍♂️

Reply:
Good point, I guess this will not copy artifact creatures you control with Shroud ? Will test this if home from work